src/js/components/App.js

```diff
@@ -209,38 +208,34 @@ class App extends Component {
     }
 
     return (
-      <Spring native from={{ opacity: 0 }} to={{ opacity: 1 }} config={config.stiff}>
```

Comment:
react-spring dependency was removed. Fade in animation is done via CSS now. There was an conflict with the Content Security Policy and react-spring, which was using and `eval`.